dev1/class.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+class User{
+    protected $height;
+    protected $weight;
```

Comment:
as PHP 8.0 docs, I strongly recommend that use data type for properties and methods.